users/user-guide-query/how-to-handle-unique-counting.md

```diff
@@ -26,11 +26,33 @@ Functions:
 
 * **DistinctCountHLL**(x)\_ -> LONG\_
 
-For column type **INT**/**LONG**/**FLOAT**/**DOUBLE**/**STRING** , Pinot treats each value as an individual entry to add into HyperLogLog Object, then compute the approximation by calling method **cardinality()**.
+For column type **INT**/**LONG**/**FLOAT**/**DOUBLE**/**STRING** , Pinot treats each value as an individual entry to add into HyperLogLog Object, then compute the approximation by calling method _**cardinality()**.
 
 For column type **BYTES**, Pinot treats each value as a serialized HyperLogLog Object with pre-aggregated values inside. The bytes value is generated by `org.apache.pinot.core.common.ObjectSerDeUtils.HYPER_LOG_LOG_SER_DE.serialize(hyperLogLog)`.
 
-All deserialized HyperLogLog object will be merged into one then calling method _\*\*cardinality() \*\*to get the approximated unique count._
+All deserialized HyperLogLog object will be merged into one then calling method _**cardinality()** to get the approximated unique count._
+
+### HyperLogLogPlusPlus
+
+The [HyperLogLog++](https://en.wikipedia.org/wiki/HyperLogLog) algorithm proposes several improvements in the HyperLogLog algorithm to reduce memory requirements and increase accuracy in some ranges of cardinalities.
+
+* 64-bit hash function is used instead of the 32 bits used in the original paper. This reduces the hash collisions for large cardinalities allowing to remove the large range correction.
```

Comment:
in the original algorithm?

Reply:
would this be accurate:
This reduces the hash collisions for large cardinalities, removing the large range correction.

Reply:
the wiki link for HyperLogLog https://en.wikipedia.org/wiki/HyperLogLog, it has a section pointing to HyperLogLog++ algorithm. the rephrase looks ok.